src/util/SpawnPed.cpp

```diff
@@ -0,0 +1,50 @@
+#include "core/frontend/Notifications.hpp"
+#include "game/backend/FiberPool.hpp"
+#include "game/backend/ScriptMgr.hpp"
+#include "game/rdr/Natives.hpp"
+
+
+namespace YimMenu
+{
+	bool SpawnPed(std::string model_name, Player player, int playerPed)
+	{
+		FiberPool::Push([=] {
+			Hash model = MISC::GET_HASH_KEY(model_name.c_str());
```

Comment:
use `IS_MODEL_VALID` to verify the model.

Reply:
I already check that if it is in the cd image. the validity should be checked first is what you are saying right?

Reply:
Overlooked it.